modules/install/pages/install-ports.adoc

```diff
@@ -45,7 +45,7 @@ These paths are defined as:
 As of Couchbase Server Version 7.0, the XDCR protocol Version 2 (_XMEM_), which uses the Memcached Binary protocol, is the only XDCR protocol supported.
 Version 1 (_CAPI_), which used the REST protocol, is no longer supported.
 Refer to xref:xdcr-reference:xdcr-advanced-settings.adoc[XDCR Advanced Settings] for information about the XDCR protocol version and other advanced settings.
-
+* _cbbackupmgr_ (backup client): This Couchbase backup client connects to the ports on the node that it requires access to. +
```

Comment:
Realized that the previous description I had given was not correct. It should say:

* _cbbackupmgr_ (backup client): This Couchbase backup client connects to the cluster services using the ports for the service. +

Also, there should be a blank line after this bullet item since the last line (current line 49) was previously separated by a blank line from the bulleted items.